cross/libvpx/Makefile

```diff
@@ -0,0 +1,53 @@
+PKG_NAME = libvpx
+PKG_VERS = 1.5.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/webmproject/libvpx/archive
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = http://www.webmproject.org/code/
+COMMENT  = WebM VP8/VP9 Codec
+LICENSE = https://raw.githubusercontent.com/webmproject/libvpx/master/LICENSE
+
+INSTALL_TARGET = myInstall
+
+include ../../mk/spksrc.cross-cc.mk
```

Comment:
If I move it after the `ifeq` conditions, the results will be always false because `ARM5_ARCHES` is defined in the include
So I can move it between `YASM_PATH = ...` and `ifeq ($(findstring $(ARCH),$(ARM5_....`.